js/gmcqView.js

```diff
@@ -0,0 +1,20 @@
+import McqView from 'components/adapt-contrib-mcq/js/mcqView';
+
+class GmcqView extends McqView {
+
+  setupQuestion(...args) {
+    super.setupQuestion(...args);
+  }
+
+  onQuestionRendered() {
+    this.$('.js-item-label').imageready(() => {
+      this.setReadyStatus();
+    });
+
```

Comment:
Could change this to a snazzy one liner
`this.$('.js-item-label').imageready(() => this.setReadyStatus());`

Reply:
Snazz incorporated, thanks!